src/object.c

```diff
@@ -1210,6 +1210,7 @@ struct redisMemOverhead *getMemoryOverheadData(void) {
             raxSize(server.repl_backlog->blocks_index) * sizeof(void*);
     }
     mem_total += mh->repl_backlog;
+    mem_total += server.repl_full_sync_buffer.mem_used;
```

Comment:
this change isn't listed in the top comment

Reply:
added